### ISSUE TITLE
custom partial body-aftercontent.html doesn't work as expected

### ISSUE THREAD

**Issue by eyoder** [vjeantet/hugo-theme-docdock#134]
I duplicated this file, added it to my /layouts/partials folder, and edited a few things, but the changes don't have any affect. This only works if I change the themes/layouts/partials file.
Suggestions?

**Comment by rdev0rigin** [comment]
I am having a similliar issue when i try to overload the ´childs.html´ shortcode.

**Comment by Pilskalns** [comment]
Hi @eyoder, @rdev0rigin 

Does restarting Hugo server helps to this issue? Does final build also have it?

Sometimes editing core layout files it produces odd results with overflowed TOC tables etc. Restarting server usually helps, I guess this is a habit for me already, though in recent Hugo versions this have greatly improved.

**Comment by rdev0rigin** [comment]
@Pilskalns No, restarting the server doesn't work, Im not sure about the final build.  

What I discovered is that you cannot define two templates with the same name. For example, in the `children.html` shortcode it defines a template `{{ define "childs" }}` and also executes it `{{- template "childs" ... }}`.

If you cut and paste the `children.html` shortcode into your `project/layouts/shortcodes/` dir then and edit it, if the templates definition is still named `"childs"` then the themes definition overwrites your projects definition.

**Comment by Pilskalns** [comment]
I guess this is more to the how Hugo template/variable lookup order works.

I've never heard of, but there must be some parents in the world who have named their both children in the same name...

Using distinctive variables would be a good remedy here?
